lib/msal-node/src/network/HttpClient.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INetworkModule, NetworkRequestOptions } from '@azure/msal-common';
+import { HttpMethod } from './../utils/Constants';
+import axios from 'axios';
+
+/**
+ * This class implements the Fetch API for GET and POST requests. See more here: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
+ */
+export class HttpClient implements INetworkModule {
+    /**
+     * Axios library for REST endpoints - Get request
+     * @param url
+     * @param headers
+     * @param body
+     */
+    async sendGetRequestAsync<T>(url: string, options?: NetworkRequestOptions): Promise<T> {
+
+        // axios config
+        const request = {
+            method: HttpMethod.GET,
+            url: url,
+            headers: (options && options.headers)
+        };
+
+        // GET call
+        const response = await axios(request);
+        return await response.data as T;
```

Comment:
Don't think you need to `await` here. Same thing below.

Reply:
Done, sorry carry over from the template code in `msal-browser`